apps/docs-website/docs/vscode-support.md

```diff
@@ -19,7 +21,17 @@ To make the [VSCode ESLint Extension](https://marketplace.visualstudio.com/items
 }
 ```
 
-### Astro support
+## `eslint.config.ts` support
+
+If you are using the `eslint.config.ts` config, add this to your `.vscode/settings.json`:
+
+```JSONC title=".vscode/settings.json"
+"eslint.options": {
+  "flags": ["unstable_ts_config"]
+}
+```
+
+## Astro support
 
```

Comment:
👎

please don't move this section, it makes diffs harder to review

Reply:
The diff showed that I moved it before, so I moved it back to make it _easier_ to review. Hmph. I need to read it.